platforms/hyperledger-besu/configuration/README.md

```diff
@@ -1,49 +1,49 @@
-6. Edited and saved network.yaml with all fields populated according to your requirements. A Sample network.yaml can be found [here](./samples/network-besu.yaml)
+## Configuration pre-requisites
+For each organization in the DLT network you need to set up the following:
+1. One Managed Kubernetes cluster; BAF is currently tested on Amazon EKS, which means you will need AWS CLI set up as well.
+2. A Hashicorp Vault installation for each organization which is initialized and unsealed. The Vault Address should be accessible from this machine (where the playbook is run) and the Kubernetes cluster. The Vault root token is used in the network configuration, so this should be available as well.
+3. A Git User with write access to all the branches in the chosen Git repository; as well as an access token.
+4. The network configuration file (`network.yaml`) which has been filled in according to your requirements. A sample `network.yaml` for Hyperledger Besu can be found [here](./samples/network-besu.yaml).
 
 ## Execution 
-
-**NOTE**: If you are running the playbooks independently i.e. not using platforms/shared/configuration/site.yaml, then use the same network.yaml as input for each playbook.
-
 ### Step 1
-Ensure that the `network.yaml` is edited properly and saved. Follow the guidance in the comments and readme.
+Ensure that the `network.yaml` is edited properly and saved. Follow the guidance in the comments (from the sample file found [here](./samples/network-besu.yaml)).
 
 ### Step 2
-Ideally, the configuration should be run by the following command from the root of the project. The [platforms/shared/configuration/site.yaml](../../shared/configuration/site.yaml) is the master playbook which does basic environment setup, kubernetes environment setup and then calls platform specific deployment playbooks.
+Execute the playbook by running the command below - executed from the root of the project:
 ```
-ansible-playbook platforms/shared/configuration/site.yaml -e "@./platforms/samples/network-besu.yaml"
+ansible-playbook platforms/shared/configuration/site.yaml -e "@/path/to/network-besu.yaml"
 ```
-Ensure that you have added an **SSH Key** with read-write permission to your git repository before executing the playbook. Follow steps in [README](../../shared/configuration/README.md).<br>
-If you just want to run this deploy-network.yaml playbook, then, from the *platforms/Hyperledger Besu/configuration* directory, run the following command for deploying the network
+The [platforms/shared/configuration/site.yaml](../../shared/configuration/site.yaml) is the master playbook which does basic environment setup, configures the Kubernetes cluster and then calls platform specific deployment playbooks.
+
+You can also only run the platform specific deployment playbooks by running the command below - executed from the root of the project:
```

Comment:
Add "after the prerequisites have been installed"

Reply:
Resolved in commit [9f27d09](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/9f27d09fc1063e42936f0cdc81410dc63f7106e9)